README.md

```diff
@@ -385,6 +386,26 @@ steps:
       # ...
 ```
 
+### Skip rosdep install
+
+Include an option to bypass `rosdep install` for workflow that uses specific docker image and better control of dependencies. To check for missing dependencies within the workflow's image, user can run with `rosdep-check: true` flag.
+
+```yaml
+runs-on: ubuntu-latest
+container:
+  image: rostooling/setup-ros-docker:ubuntu-jammy-ros-iron-ros-base-latest
+steps:
+  # ...
+  - uses: ros-tooling/action-ros-ci@v0.3.6
```

Comment:
We just use the minor version tag (`v0.3`) in README examples:

```suggestion
  - uses: ros-tooling/action-ros-ci@v0.3
```

`v0.3.6` doesn't exist anyway (`0.3.6` does).

Reply:
Committed the suggestion. Just for clarification, I think the `version` in this example should match with whichever version it is potentially released to, so I assumed it will be `v0.3` in this case?

Reply:
Indeed! It doesn't break existing `v0.3` workflows because it's a new feature, so I'll just release it as `v0.3`/`0.3.7`.